src/components/primary-button/PrimaryButton.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import Button, { ButtonProps } from '../button';
+import Button, { ButtonProps } from '../button/Button';
```

Comment:
I thought we were moving toward using an implicit import pattern via `index.js` rather than the component module. Is that no longer possible due to Typescript support?

Reply:
Its possible like it was working yesterday, but requires `export * from foo` style in index.js, which we can't have due to rewire in parent repos